users/edgar/default.nix

```diff
@@ -10,8 +12,7 @@
       home = "/home/edgar";
       description = "Edgar Saldivar";
       extraGroups = [ "wheel" "networkmanager" ];
-      hashedPassword = secrets.users.edgar.hashedPassword;
-      #password = "123";
+      hashedPasswordFile = config.sops.secrets."users/edgar/hashedPassword".path;
```

Comment:
this references `config` from the inputs, which contains **all** config that is _ever_ defined in _any_ file. The `sops.secrets…` above makes its way in to the global `config` eventually, and we reference it here.

Reply:
the `path` portion is created at build-time by the sops-nix module.